util/chunk/chunk.go

```diff
@@ -609,6 +609,19 @@ func (c *Chunk) Column(colIdx int) *Column {
 	return c.columns[colIdx]
 }
 
+// SetCol sets the colIdx Column to col and returns the old Column.
+func (c *Chunk) SetCol(colIdx int, col *Column) *Column {
+	for len(c.columns) <= colIdx {
```

Comment:
When will this happen?

Reply:
After reviewing our `Projection` executor again, it doesn't happen, so I remove these codes.